cves/kernel/CVE-2020-12464.yml

```diff
@@ -75,7 +78,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: ["https://bugzilla.redhat.com/show_bug.cgi?id=1831726"]
```

Comment:
I think you should populate this field with the bug IDs that are involved.

> [1831731](https://bugzilla.redhat.com/show_bug.cgi?id=1831731) [1836894](https://bugzilla.redhat.com/show_bug.cgi?id=1836894) [1836895](https://bugzilla.redhat.com/show_bug.cgi?id=1836895) [1836896](https://bugzilla.redhat.com/show_bug.cgi?id=1836896) [1836897](https://bugzilla.redhat.com/show_bug.cgi?id=1836897) [1836898](https://bugzilla.redhat.com/show_bug.cgi?id=1836898) [1870321](https://bugzilla.redhat.com/show_bug.cgi?id=1870321) [1900751](https://bugzilla.redhat.com/show_bug.cgi?id=1900751)

https://bugzilla.redhat.com/show_bug.cgi?id=1831726

Reply:
thanks for finding those! didn't realize there's more - I could only open the first one, the others are giving me 'access denied' so I will just add the first one